aws-lc-rs/src/iv.rs

```diff
@@ -0,0 +1,98 @@
+// Copyright 2018 Brian Smith.
+// SPDX-License-Identifier: ISC
+// Modifications copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+#![allow(dead_code)]
+
+//! Initialization Vector (IV) cryptographic primitives
```

Comment:
This module could also be a separate PR, along with using it everywhere that isn't the CBC/CTR stuff